tests/test_tls.py

```diff
@@ -0,0 +1,35 @@
+import logging
+import pytest
+
+import spark_utils as utils
+
+log = logging.getLogger(__name__)
+
+@pytest.fixture(scope="module", autouse=True)
+def setup_spark(configure_security_spark, configure_universe):
+    service_options = {
+        "service": {
+            "tls": {
+                "enabled": True,
+                "protocol": "TLSv1.2",
```

Comment:
where is this protocol used?  I see you are adding this to the in this PR `config.json` but I don't see where it is used

Reply:
It is actually one of the protocol supported by JVM and it is the latest one. We can also give other versions if they are supported by particular JVM. In Spark, we need to specify this protocol in case of TLS enabled.